lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -378,7 +378,7 @@ function paraneue_dosomething_get_themed_reportback_item($data, $rogue_rb_item =
         );
       }
 
-      $data->kudos = dosomething_kudos_get_kudos_data_for_fid($data->id, $total_rogue_kudos ? $total_rogue_kudos : 0, $liked_by_current_user ? $liked_by_current_user : null);
+      $data->kudos = dosomething_kudos_get_kudos_data_for_fid($data->id, $rogue_rb_item ? $total_rogue_kudos : 0, $liked_by_current_user ? $liked_by_current_user : null);
```

Comment:
oh good call! I think we still want to use `$liked_by_current_user` but I deleted `$total_rogue_kudos`!